easel/lib/easel_provider.dart

```diff
@@ -814,12 +813,27 @@ class EaselProvider extends ChangeNotifier {
       uploadThumbnailResponse = uploadResponse.getOrElse(() => StorageResponseModel.initial());
     }
 
-    final response = await repository.uploadFile(
-      file: _file!,
-      onUploadProgressCallback: (value) {
-        _uploadProgressController.sink.add(value);
-      },
-    );
+    final whereToUpload = QuickNode.listOfQuickNodeAllowedExtension().contains(fileExtension.toLowerCase());
+
+    Either<Failure, StorageResponseModel> response;
+
+    if (!whereToUpload) {
+      response = await repository.uploadFile(
+        file: _file!,
+        onUploadProgressCallback: (value) {
+          _uploadProgressController.sink.add(value);
+        },
+      );
+    } else {
+      response = await repository.uploadFileUsingQuickNode(
+        uploadIPFSInput: UploadIPFSInput(
```

Comment:
Does quick node doesn't provides any support for the upload progress? Kindly check line 824.

Reply:
yes it is supported working on it